### PR TITLE
Translate configuration views labels

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Tooltips for dnsentries: https://github.com/gardener/external-dns-management/blob/9fd844dcc0970a1cc65a22b572f6dc41441da26f/docs/usage/dnsentry_status.md
- Tooltips for dnsproviders: https://github.com/gardener/external-dns-management/blob/9fd844dcc0970a1cc65a22b572f6dc41441da26f/docs/usage/dnsprovider_status.md
- Tooltips for Issuers: https://github.com/gardener/cert-management/blob/52eb685e423c6ea0f9aeaf4c1524ce417dbcc709/pkg/apis/cert/v1alpha1/issuer.go#L123
- I didn't find any valuable states for the certificates

**Related issue(s)**

 #658

### CODE REVIEW

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -649,6 +653,10 @@ dnsentries:
     ttl: Enter the time to live.
   title: DNS Entries
   tooltips:
+    deleting: The deletion of the DNS records is in progress.
+    invalid: There is a conflict with another DNS entry or owner.
+    pending: The update of the DNS entry is batched or in progress.
```
Comment (grego952) [inline core/src/i18n/en.yaml:658]:
```suggestion
    pending: The DNS entry update is batched or in progress.
```

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -649,6 +653,10 @@ dnsentries:
...
+    stale: The DNS entry is existing but there is a problem with the provider.
```
Comment (grego952) [inline core/src/i18n/en.yaml:659]:
```suggestion
    stale: The DNS entry exists, but there is an issue with the provider.
```

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -1006,6 +1014,7 @@ issuers:
     included-domains: The domains (including subdomains) for which certificate requests are allowed, $t(domains.example).
     issuer-type: Certification authority used
     key-secret-ref: keySecretRef is the secret ref to the Secret that holds the symmetric MAC key of the External Account Binding with data key 'hmacKey'. The secret key stored in the Secret **must** be un-padded, base64 URL encoded data.
+    pending: The update of the Issuer is batched or in progress.
```
Comment (grego952) [inline core/src/i18n/en.yaml:1017]:
```suggestion
    pending: The Issuer's update is batched or in progress.
```

**Review by Wawrzyn321 — COMMENTED** [review]
I see
image
but I suppose it will be done [in other task](https://github.com/kyma-project/busola/issues/660).

File: core-ui/src/components/Predefined/Details/AddonsConfiguration.details.js [hunk 4]
```diff
@@ -12,16 +12,7 @@ const RepositoryUrls = addon => {
...
-      resourceKind="addons"
-      ariaLabel={t('addons.addons-status')}
-      additionalContent={repo.message}
-      autoResolveType
-      i18n={i18n}
-      noTooltip={repo.status === 'Ready'}
-    >
-      {repo.status}
-    </StatusBadge>,
+    <ResourceStatus status={repo} resourceKind="repositories" i18n={i18n} />,
```
Comment (Wawrzyn321) [inline core-ui/src/components/Predefined/Details/AddonsConfiguration.details.js:15]: There's no `repositories` key in en.yaml file.

**Review by Wawrzyn321 — CHANGES_REQUESTED** [review]
Colons are duplicated for some error messages - `common.tooltips.error` returns 'Error: '.
image

**Review by Wawrzyn321 — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Pull request merged**